src/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { filterPeople } from '../utils/filteredPeople';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState(false);
+
+  const [searchParams] = useSearchParams();
+  const centuries = searchParams.getAll('centuries') || [];
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
```

Comment:
move all these query param names to some enum and reuse them here, this will make it easier to change the parameter name, or add a new one, and reduce the number of errors